lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1814,7 +1814,7 @@ namespace Js
             // Free unused bytes
             Assert(cbNeeded + 1 <= cbUtf8Buffer);
             *ppSourceInfo = Utf8SourceInfo::New(this, utf8Script, (int)length,
-                cbNeeded, pSrcInfo, isLibraryCode, scriptSource);
+                cbNeeded, pSrcInfo, isLibraryCode, nullptr);
```

Comment:
One additional comment I had was, it would be good to leave comments on why we don't need to pass in the parameter here so that people looking at the code later aren't confused